service/api/persons.py

```diff
@@ -1,13 +1,13 @@
 from uuid import UUID
+
 from flask import abort, jsonify
-from webargs.flaskparser import use_args
 from marshmallow import Schema, fields
+from webargs.flaskparser import use_args
 
-from service.server import app, db
 from service.models import Person
+from service.server import app, db
 
 
-###### Validation Schemas ######
```

Comment:
I would leave these comments.  I think the might be helpful to someone who might not be familiar with this pattern.